lang/py3/avro/tests/run_tests.py

```diff
@@ -69,9 +59,16 @@ def SetupLogging():
   console_handler.setLevel(logging.DEBUG)
   logging.root.addHandler(console_handler)
 
-
 SetupLogging()
 
+from avro.tests.test_datafile import *
+from avro.tests.test_datafile_interop import *
+from avro.tests.test_io import *
+from avro.tests.test_ipc import *
+from avro.tests.test_protocol import *
+from avro.tests.test_schema import *
+from avro.tests.test_script import *
+from avro.tests.test_enum import *
```

Comment:
Why move these imports? `isort` will disagree with this change, and break the changes going into #532

Reply:
Riiight, I see the problem. Maybe a compromise (until we can do better test discovery in this case) would be to wrap the imports in a `try/except ImportError` block?

Reply:
Thanks for the review @kojiromike! Considering your comment, I reverted the previous change and wrapped the try/except block into a function to defer its evaluation after the logging setup.